tests/graph_rust.rs

```diff
@@ -0,0 +1,53 @@
+use graph_rust::{Bfs, Dfs, Graph, GraphType};
+
+#[test]
+fn bfs_non_rec_test() {
+    let g = Graph::default()
+        .with_dir(GraphType::Directed)
+        .with_schema(String::from("..."))
+        .from_file(String::from("data_edges.csv"), None, ",")
+        .build();
+
+    g.bfs(2);
+    println!()
+}
+
+#[test]
+fn dfs_non_rec_test() {
+    let g = Graph::default()
+        .with_dir(GraphType::Directed)
+        .with_schema(String::from("..."))
+        .from_file(String::from("data_edges.csv"), None, ",")
+        .build();
+
+    g.dfs(1);
+    println!()
+}
+
+#[test]
+fn dfs_iter_test() {
+    let g = Graph::default()
+        .with_dir(GraphType::Directed)
+        .with_schema(String::from("..."))
+        .from_file(String::from("data_edges.csv"), None, ",")
+        .build();
+    let mut dfs = Dfs::new(&g, 1);
+    while let Some(node) = dfs.next() {
+        print!("{}->", node);
+    }
+    println!()
+}
+
+#[test]
+fn bfs_iter_test() {
+    let g = Graph::default()
+        .with_dir(GraphType::Directed)
+        .with_schema(String::from("..."))
+        .from_file(String::from("data_edges.csv"), None, ",")
```

Comment:
Do not just use one graph data. Suppose you can use different types of graph data for testing. For that purpose, you may want to construct the graph in codes, rather than always reading from file. In addition, reading from files should be independently tested.